src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00ACDC;
+}
+
+
+.header {
+  box-sizing: border-box;
+  width: 100vw;
+  padding: 0 50px;
+  background-color: #fff;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav_link {
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+
+  margin: 0 10px;
+  height: 60px;
+
+  display: flex;
+  flex-direction: column;
```

Comment:
you don't need to change direction, its unnecessary here, if you remove it you won't see any changes